pilot/pkg/xds/gen2.go

```diff
@@ -174,11 +109,8 @@ func (s *DiscoveryServer) handleCustomGenerator(con *Connection, req *discovery.
 		return err
 	}
 	apiPushes.Increment()
-	w.LastSent = time.Now()
```

Comment:
May be we should move this to `con.send` and update there?

Reply:
done